qutebrowser/browser/browsertab.py

```diff
@@ -427,13 +427,20 @@ def reapply(self) -> None:
         self._set_factor_internal(self._zoom_factor)
 
 
+SelectionState = enum.Enum('SelectionState', [
+    'NONE',
+    'NORMAL',
+    'LINE',
+])
+
+
 class AbstractCaret(QObject):
 
     """Attribute ``caret`` of AbstractTab for caret browsing."""
 
     #: Signal emitted when the selection was toggled.
     #: (argument - whether the selection is now active)
```

Comment:
This is outdated now (probably fine to just remove this line, as the type already makes it clear what it is)